libraries/shared/factories/event/transformer_test.go

```diff
@@ -80,25 +79,15 @@ var _ = Describe("Transformer", func() {
 	})
 
 	It("returns an error if converter fails", func() {
-		converter.ToEntitiesError = fakes.FakeError
+		converter.ToModelsError = fakes.FakeError
 
 		err := t.Execute(logs)
 
 		Expect(err).To(HaveOccurred())
 		Expect(err).To(MatchError(fakes.FakeError))
```

Comment:
I think this test might now be exactly the same as the one below on lines 90-97 ✂️